.github/workflows/ci-test.yml

```diff
@@ -121,6 +121,7 @@ jobs:
           path: .coverage.integration${{ matrix.branch }}
 
   coverage:
+    if: !(github.event_name == 'schedule')
```

Comment:
This setting is not working, not sure if it needs to be quoted or what. See link below for error.

https://github.com/phac-nml/irida-uploader/actions/runs/4159822410/workflow